host/src/main.rs

```diff
@@ -11,6 +10,10 @@ use tracing_subscriber::FmtSubscriber;
 
 #[tokio::main]
 async fn main() -> HostResult<()> {
+    // defailt `info` level if nothing else defined
+    if env::var("RUST_LOG").is_err() {
+        env::set_var("RUST_LOG", "info")
+    }
```

Comment:
Seems already set to the default value here: https://github.com/taikoxyz/raiko/blob/a1ee3ae56704658807ced7aa5fef10b56df3986a/host/src/lib.rs#L97-L100 @petarvujovic98?

Reply:
Yes, it is already the default and used here https://github.com/taikoxyz/raiko/blob/a1ee3ae56704658807ced7aa5fef10b56df3986a/host/src/main.rs#L32-L38

Reply:
I dont believe that approach works (or is triggered). Will double check.

Reply:
it does work, some logs were trying to output before init was happening. re-shuffled and reverted my init. Thanks.